src/registrar/admin.py

```diff
@@ -1698,7 +1698,7 @@ def get_fieldsets(self, request, obj=None):
             for name, data in fieldsets:
                 fields = data.get("fields", [])
                 fields = tuple(field for field in fields if field not in self.superuser_only_fields)
-                modified_fieldsets.append((name, {"fields": fields}))
+                modified_fieldsets.append((name, {**data, "fields": fields}))
```

Comment:
Just out of curiosity, what does adding **data as a param do when appending to our updated fieldsets, and what makes this case different from the other show/hide detail collapses?

Reply:
This is a SUPER good question, and honestly it was like 23842834 blog posts discovery later that I found this and this was also best explanation:

* Without `**data`: It's like taking an item (the fields) out of a box and putting it in a new box without the original settings (like collapsibility), so we lose the settings we have on the fields

* With `**data`: It's like taking the items out of the box carefully, changing just the fields, and putting everything back in the same box and then this way all the original settings (like collapsibility) stay intact